src/js/control-bar/time-controls/time-display.js

```diff
@@ -66,11 +66,26 @@ class TimeDisplay extends Component {
     if (this.textNode_) {
       this.contentEl_.removeChild(this.textNode_);
     }
-    this.textNode_ = document.createTextNode(` -${this.formattedTime_ || '0:00'}`);
+    this.textNode_ = document.createTextNode(this.formattedTime_ || '0:00');
```

Comment:
This was the relevant mistake. Not all sub-classes wanted to display negative formatted times.